common/src/main/java/net/dodogang/plume/Plume.java

```diff
@@ -43,30 +45,30 @@ public static void initialize() {
         if (runDevTests) {
             LOGGER.log(Level.INFO, "Development environment detected! Running dev initialize.");
 
-            ItemGroup itemGroup = new TabbedItemGroup("plume_test",
-                (id) -> {
-                    String ns = id.getNamespace();
-                    return ImmutableList.of(
-                        TabbedItemGroup.createTab(Blocks.DIRT, ns, "unga"),
-                        TabbedItemGroup.createTab(Blocks.BLACK_CONCRETE, ns, "bunga")
-                    );
-                }, () -> new ItemStack(Blocks.STONE)
-            );
+            ItemGroup itemGroup = new TabbedItemGroup("plume_test", (id) -> {
+                String ns = id.getNamespace();
+                return ImmutableList.of(
+                    TabbedItemGroup.createTab(Blocks.DIRT, ns, "unga"),
+                    TabbedItemGroup.createTab(Blocks.BLACK_CONCRETE, ns, "bunga"));
+            }, () -> new ItemStack(Blocks.STONE));
 
-            BlockRegistryBatch blocks = new BlockRegistryBatch(MOD_ID).setDefaultItemSettings(new Item.Settings().group(itemGroup));
+            BlockRegistryBatch blocks =
+                new BlockRegistryBatch(MOD_ID).setDefaultItemSettings(new Item.Settings().group(itemGroup));
```

Comment:
I don't like this. I'd rather it be:
```java
BlockRegistryBatch blocks = new BlockRegistryBatch(MOD_ID)
        .setDefaultItemSettings(new Item.Settings().group(itemGroup));
```
I don't know if that's possible tho

Reply:
for builders this short (one 'argument') I prefer these as one line, personally